aepp/configs.py

```diff
@@ -367,3 +370,141 @@ def setSandbox(self,sandbox:str=None)->dict:
         self.header["x-sandbox-name"] = sandbox
         self.__configObject__["sandbox"] = sandbox
         return self.getConfigObject()
+
+    def setOauthV2setup(self,credentialId:str=None,orgDevId:str=None)->bool:
```

Comment:
can we use python style conventions in functions, namely with underscores for functions

Reply:
I would have agreed but the whole package is using the camelCase notation. 
I would think that we should be consistent over the different modules and keep camelCase usage. 
It does not hurt in the end.